README.md

```diff
@@ -211,46 +207,36 @@ This last step will show you how to create a GitHub Issue to track the work that
 
-Now that both of these alerts are back to an "Open" state, we can come up with a plan to fix them.  If you look at the alerts, they are both calling out one specific file containing the issues: `server/routes.py`.  The issue is in how we're crafting the SQL query for the database.  These queries are vulnerable to SQL injection attacks.  We need to get these SQL statements re-written more securely.  
+Now that both of these alerts are open, let's fix them. If you look at the alerts, they both call out one specific file containing the issues: `server/routes.py`. The issue is in crafting the SQL query for the database. These queries are vulnerable to SQL injection attacks. We should rewrite these SQL statements more securely. 
   
-If you expand the **More info** section at the bottom of the alert, there are very clear suggestions to fix this query.  We're going to implement those suggestions in the next activity.
+If you expand the **More info** section at the bottom of the alert, there are very clear suggestions to fix this query. We're going to implement those suggestions in the next activity.
 
 ### :keyboard: Activity 2: Edit routes.py
-Now that we know where the issues exist and how to fix them, we're going to modify the file `routes.py` to fix the problems.  Again, you'll want to do these next steps in a separate browser window or tab.
-  
-1. In the **Code** pane, click on the folder named `server`, then click on the file named `routes.py`.  The code view page will open.
-  
-  ![image](https://github.com/leftrightleft/enable-code-scanning/assets/4910518/2af87da0-b39a-469c-ab37-d3121660978e)
-  
-2. Click the edit button
-  
-  ![image](https://github.com/leftrightleft/enable-code-scanning/assets/4910518/ac53e02c-a169-439f-b04a-37c36048514d)
-  
-3. Edit line 16 by highlighting the SQL statement and replace it with this text: `"SELECT * FROM books WHERE name LIKE %s", name`.  Your page should look like this:
-  
-  ![image](https://github.com/leftrightleft/enable-code-scanning/assets/4910518/0513c535-a4a1-4be4-b7e0-efb0ed49827e)
+We now know where the issues exist and how to fix them. We'll start by modifying the file `routes.py`. Again, you'll want to do these next steps in a separate browser window or tab.
   
-4. Next we're going to edit line 22.  We'll replace the SQL statement with this text: `"SELECT * FROM books WHERE author LIKE %s", author`
+1. Click the **Code** tab in your repository.
+2. Select the `server` folder.
+3. Select the `routes.py` file.
+4. Click the **Edit** button to the right.
   
-  ![image](https://github.com/leftrightleft/enable-code-scanning/assets/4910518/a5f7b863-aa06-4302-846b-82af27f1fa84)
+  ![edit-button.png](images/edit-button.png)
   
-5. Select **Commit changes...** from the top right 
+5. Edit line 16 by highlighting the SQL statement and replace it with this text: `"SELECT * FROM books WHERE name LIKE %s", name`.
```

Comment:
This is 100% my own ignorance in this language, but should there be more information as to why these changes are being made? Or is it assumed those taking this course will know why the `+ "%'"` is removed?

Reply:
I think we explain this pretty well on line [220](https://github.com/leftrightleft/enable-code-scanning/pull/6/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R220).  I don't think we need to go much more into the details here.

Reply:
Awesome! Thank you for pointing that out 😄